src/main/java/com/laytonsmith/core/functions/Minecraft.java

```diff
@@ -922,6 +929,23 @@ public Construct exec(Target t, Environment env, Construct... args) throws Cance
 				//Server Ip
 				retVals.add(new CString(server.getIp(), t));
 			}
+			if (index == 13 || index == -1) {
+				//Uptime
+				double uptime = (double)(System.currentTimeMillis() - ManagementFactory.getRuntimeMXBean().getStartTime());
+				retVals.add(new CDouble(uptime, t));
```

Comment:
I didn't notice this before, but why is this a double and CDouble instead of long and CInt?